spring-examples/spring-session/simple-spring-session-client-server/README.md

```diff
@@ -0,0 +1,45 @@
+# Simple Spring Session Client-Server Example
+
+## Description
+This example is a simple Spring Session example, where there is a GemFire client initialized within a Spring Boot web app, that will support session information management. This example is setup to expire session information within 30 second of it being committed. This 30 second expiration is for demonstration purposes only.
+
+## Prerequisites
+In order to run the following example one requires the following to be set up:
+* Java 17
+* Latest VMware GemFire is installed locally
+* Set up local `gradle` or `maven` environment by following Steps 1-6 and 8 from [Spring Boot For VMware GemFire Quickstart](https://docs.vmware.com/en/Spring-Boot-for-VMware-GemFire/index.html#spring-boot-for-vmware-gemfire-quick-start-0)
+
+## Steps to run
+1. Clone `Simple Spring Session Client Server` project
+2. Start GemFire server-side processes 
+   1. Open new Terminal
+   2. `cd` into the `Simple Spring Session Client Server` project directory
+   3. Start `gfsh` located in the `{VMware_GemFire_installation_location}/bin` directory
+   4. Locate the `start.gfsh` file in the `Simple Spring Session Client Server` project
```

Comment:
I think we can remove step 4.  It's repeated in step 5 already.  Or remove the wording from step 5